user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md

```diff
@@ -27,3 +29,81 @@ Please note the following regarding suggestion events:
   - From DataMiner 10.4.0 [CU1]/10.4.4 onwards<!-- RN 38674 -->, at most 50 new suggestion events per hour per type of anomaly (i.e. level shift, trend change, flatline, or variance change) can be generated per DataMiner Agent. There is no limit to the maximum number of anomaly alarm events, i.e. alarm events for parameters that have explicit [anomaly alarm monitoring configured in the alarm template](xref:Configuring_anomaly_detection_alarms).
   - From DataMiner 10.2.11/10.3.0 onwards, suggestion events are only created for the most significant changes, and per hosting DataMiner Agent there can be at most 500 open suggestion events related to behavioral anomaly detection.
   - Prior to DataMiner 10.2.11/10.3.0, suggestion events are created for all anomalous behavioral changes that do not have alarm monitoring enabled.
+
+## Providing feedback on behavioral anomalies
```

Comment:
I wonder if it might not be better to add this section on a separate page. I know we're being a bit careful with adding new pages because of the page limit, but in this case I do think it would be a definite improvement.

Reply:
Agreed. I've added it as a separate page in [commit 06d609b](https://github.com/SkylineCommunications/dataminer-docs/pull/3761/commits/06d609bfe9d538af003719998837c77037d88f47).